recipes/salty/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "1.0.0" %}
+{% set name = "salty" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/LanLab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: f849d50e83a6159a7ba48cd77faa80a4ca74e31a9db48af3cd5ab4fda5eb0b39
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+
+requirements:
+  host:
+    - python >=3.6
```

Comment:
It doesn’t really make a difference, but note that as Python 3.6 is end of life, it would make more sense to require at least 3.7.

Reply:
thank you for updating